src/mappings/entities/mapTransferManager.ts

```diff
@@ -91,19 +104,54 @@ const handleExemptionsAdded = async (blockId: string, params: Codec[]): Promise<
 
   const transferManager = await getTransferManager(ticker, transferManagerValue);
 
+  const opType =
+    transferManager.type === TransferRestrictionTypeEnum.Percentage
+      ? StatOpTypeEnum.Balance
+      : StatOpTypeEnum.Count;
+
+  const promises = [];
+
   if (transferManager) {
     transferManager.exemptedEntities = [
       ...new Set<string>([...parsedExemptions, ...transferManager.exemptedEntities]),
     ];
     transferManager.updatedBlockId = blockId;
 
-    await transferManager.save();
+    promises.push(transferManager.save());
+
+    const exemptKey = {
+      assetId: ticker,
+      opType,
+      claimType: null,
+    };
+    transferManager.exemptedEntities.forEach(entity => {
+      const upsert = async () => {
+        const exemptionId = `${ticker}/${opType}/null/${entity}`;
+        let exemption = await TransferComplianceExemption.get(exemptionId);
+        if (exemption) {
+          exemption.updatedBlockId = blockId;
+        } else {
+          exemption = TransferComplianceExemption.create({
+            id: exemptionId,
+            ...exemptKey,
+            exemptedEntityId: entity,
+            createdBlockId: blockId,
+            updatedBlockId: blockId,
+          });
+        }
+        return exemption.save();
+      };
+      promises.push(upsert());
+    });
```

Comment:
instead of re-writing this I think we should extract the method in `mapStatistics` file and reuse here